### PR TITLE
Task/a11y fixes part3

### PR DESCRIPTION
**Pull request by andrewleith**
# Summary | Résumé
This is the last of 3 PRs to fix up a11y and HTML validation issues (see commits):

- update selector for equal width table on monthly stats page
- fix incorrect legend elements in contact form
- refactor radio and select macros to not add an `aria-labelledby` attribute for no reason
- [element-permitted-content](https://html-validate.org/rules/element-permitted-content.html): remove invalid and unecessary legend element
- fix incorrect legend element
- [no-dup-class](https://html-validate.org/rules/no-dup-class.html): remove duplicate class
- [no-dup-class](https://html-validate.org/rules/no-dup-class.html): remove duplicate class

### CODE REVIEW

File: app/templates/views/templates/create.html
Comment (smcmurtry) [inline app/templates/views/templates/create.html:1]: tested w/ a screenreader and seems to work the same as before!

File: app/templates/views/contact/form.html
Comment (smcmurtry) [inline app/templates/views/contact/form.html:1]: tested the page w/ voiceover, seems to work the same as before ✅

File: app/templates/views/api/keys/create.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
   {% set button_txt = _('Create') %}
   {% call form_wrapper() %}
     {{ textbox(form.key_name, label=label_txt) }}
-    {{ radios(form.key_type, disable=disabled_options, option_hints=option_hints) }}
+    {{ radios(form.key_type, disable=disabled_options, option_hints=option_hints, use_aria_labelledby=False) }}
```
Comment (smcmurtry) [inline app/templates/views/api/keys/create.html:25]: tested w/ voiceover, works the same as before ✅

**Review by smcmurtry — APPROVED** [review]
Looks good!

File: app/templates/views/templates/create.html
Comment (andrewleith) [reply app/templates/views/templates/create.html:1]: Awesome thanks!

File: app/templates/views/contact/form.html
Comment (andrewleith) [reply app/templates/views/contact/form.html:1]: Thanks!

**Pull request merged**